TestApp/src/main/java/com/tekchand/testapp/ui/main/PlaceholderFragment.java

```diff
@@ -0,0 +1,152 @@
+package com.tekchand.testapp.ui.main;
+
+import android.app.Activity;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProviders;
+
+import com.tekchand.testapp.R;
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class PlaceholderFragment extends Fragment {
+
+    private static final String ARG_SECTION_NUMBER = "section_number";
+
+    private PageViewModel pageViewModel;
+
+    public static PlaceholderFragment newInstance(int index) {
+        PlaceholderFragment fragment = new PlaceholderFragment();
+        Bundle bundle = new Bundle();
+        bundle.putInt(ARG_SECTION_NUMBER, index);
+        fragment.setArguments(bundle);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        pageViewModel = ViewModelProviders.of(this).get(PageViewModel.class);
+        int index = 1;
```

Comment:
Why do you need to initialize this default?

Reply:
Removed unwanted things.